### DISCUSSION TITLE
DatabaseNotifications::pollingInterval(null); is not disable polling

### DISCUSSION THREAD

**Discussion by NaungYeHtet** [filamentphp/filament#10482] [Help]
In AppServiceProvider.php
````
DatabaseNotifications::pollingInterval(null);
````

In PanelProvider
````
->databaseNotifications()
````

Also ->databaseNotificationsPolling(null) is not working.

**Comment by leandrocfe** [comment]
`$panel->databaseNotificationsPolling(null)` should work. Could you share the project on github?
